resources/views/soon.blade.php

```diff
@@ -25,7 +25,7 @@
     </div>
     <div class="row">
         <div class="column">
-            Your site for fasting.
+            Your site for fasting.  Also for testing.
```

Comment:
![Womp Womp](https://m.popkey.co/d94aae/dW7GR_f-thumbnail-100-0.jpg)  
Remember not to deploy this change to production early!